Octokit.Tests.Conventions/ModelTests.cs

```diff
@@ -85,6 +85,12 @@ public void ResponseModelsHaveReadOnlyCollections(Type modelType)
                         continue;
                     }
 
+                    // Let's skip collections that only have a private Setter
+                    if (property.SetMethod != null && property.SetMethod.IsPrivate)
```

Comment:
Sorry to jump in here before this goes further off the rails, but thanks @khellang for reminding me about the intent of this test.

`ICollection<T>` has the mutable `Add`, `Remove` and `Clear` - so even if it's a read-only property, it can be manipulated arbitrarily after the object is created. We're trying to discourage that, so we use `IReadOnlyList<T>` on our collections.

An example of that is `CommitActivity` - https://github.com/octokit/octokit.net/blob/master/Octokit/Models/Response/CommitActivity.cs - if we could update `RequiredStatusChecks.Contexts` to be a more specific interface than `ICollection<T>` and then drop this change we should be able to get back to a green build...